examples/tcp-listener/tcp_listener.md

```diff
@@ -1,6 +1,6 @@
 # TCP service - Send/Receive bytes
 
-The TCP Listener is used to expose a TCP service over the TCP protocol. This sample demonstrates how the TCP socket listener service interacts with the TCP client.
+The TCP service allows you to open up a TCP socket via a `tcp:Listener`. A `tcp:Listener` is created by giving the port number. Then a TCP service is attached to the listener that accepts and serves connections from TCP clients. The `onBytes` remote method is invoked once the content is received from the client. The `onError` and `onClose` remote methods get invoked in an erroneous situation and when the connection is closed respectively. Use a TCP service to establish connections and communicate over TCP protocol or implement high level protocols based on TCP. 
```

Comment:
TCP service -> `tcp:Service`

Reply:
The TCP service allows you to open up a TCP socket via a `tcp:Listener`. A `tcp:Listener` is created by giving the port number. Then a TCP service is attached to the listener that accepts and serves connections from TCP clients.

Instead

The `tcp:Service` allows opening up a TCP socket via a `tcp:Listener`. A `tcp:Listener` is created by giving the port number, to which `tcp:Service` is attached. The listener accepts and serves connections from TCP clients.

Reply:
Updated the descriptions by [e41bbc6](https://github.com/ballerina-platform/ballerina-distribution/pull/3748/commits/e41bbc656fc78a4ed6edf0f4039f5dfe58b71119)